modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -11233,6 +11233,7 @@ old-friendly-urls=Old Friendly URLs
 old-group-name-is-a-required-system-group=Old group name is a required system group.
 old-name=Old Name
 old-role-name-is-a-required-system-role=Old role name is a required system role.
+old-value=Old value
```

Comment:
@pedro-oliveira446 could you please change to: `Old Value`? All first letters must be capitalized if it is not a sentence, that is, if it does not end with punctuation.